shell-conduit.cabal

```diff
@@ -41,6 +41,7 @@ library
                    , text
                    , transformers
                    , unix >= 2.7.0.1
+                   , exceptions
```

Comment:
Can we use exception handling code from UnliftIO.Exception instead of depending on this ? I'm assuming that should be possible by using the throw functions defined in that module.

Reply:
Seems Conduit exports the relevant things needed from the exceptions package. So, instead I used that and remove this dependency. Thanks for the contribution!